src/core/model_config_utils.cc

```diff
@@ -404,7 +404,7 @@ GetBooleanSequenceControlProperties(
     const bool required, std::string* tensor_name,
     inference::DataType* tensor_datatype, float* fp32_false_value,
     float* fp32_true_value, int32_t* int32_false_value,
-    int32_t* int32_true_value)
+    int32_t* int32_true_value, bool* bool_false_value, bool* bool_true_value)
```

Comment:
Can the function declaration be changed to taking a pointer of union of `int / float / bool` for `false_value` / `true_value`? The actual type stored can be inferred from `tensor_datatype`

Reply:
I think I can make this change. @deadeyegoodwin I would like to confirm that this can be part of the changes in implicit state management implementation.

Reply:
I think you can defer this kind of change to a later PR, be it part of implicit state or something else.  It is somewhat annoying to have so many args but I'm also not too happy about having a generic void* value.

Reply:
Got it, thanks!